src/javascript/app/pages/trade/charts/highchart.js

```diff
@@ -610,10 +606,6 @@ const Highchart = (() => {
                 const last = parseInt(last_data.x || last_data[0]);
                 if (last > (end_time * 1000) || last > ((sell_time || sell_spot_time) * 1000)) {
                     stop_streaming = true;
-                } else {
-                    // add a null point if the last tick is before end time to bring end time line into view
-                    const time = isSoldBeforeExpiry() ? (sell_time || sell_spot_time) : end_time;
-                    chart.series[0].addPoint({ x: ((time || window.time.unix()) + margin) * 1000, y: null });
```

Comment:
why not just wait for ticks to come, adding `null` point always causes some weird issues